libafl/src/state/mod.rs

```diff
@@ -262,22 +302,68 @@ pub struct StdState<I, C, R, SC> {
     /// The current index of the corpus; used to record for resumable fuzzing.
     corpus_id: Option<CorpusId>,
     stage_stack: StageStack,
+    #[cfg(feature = "dump_state")]
+    dump_state_dir: Option<PathBuf>,
     phantom: PhantomData<I>,
 }
 
+/// The standard state dump
+#[cfg(all(feature = "std", feature = "dump_state"))]
+#[derive(Serialize, Deserialize, Clone, Debug)]
+#[serde(bound = "I: serde::de::DeserializeOwned")]
+pub struct StdStateDump<I>
```

Comment:
Yeah... this makes no sense anymore - just write the state to disk using serde_json::to_string oder whatever